charts/hdfs-datanode-k8s/values.yaml

```diff
@@ -21,3 +21,11 @@
 # e.g. --set "dataNodeHostPath={/hdfs-data,/hdfs-data1}"
 dataNodeHostPath:
   - /hdfs-data
+
+# Custom hadoop configuration passed through env variables to hadoop uhopper images.
+# See https://hub.docker.com/r/uhopper/hadoop/ to get more details
+# HDFS_CONF_dfs_datanode_data_dir and CORE_CONF_fs_defaultFS need special handling and
+# they're already set by the chart so any value coming from below config will be ignored
+hadoop_env: {}
```

Comment:
I agree about the name change, yours makes more sense.

As for the comment, how about addding this phrase?
```
# Please note that these are not hadoop env variables, but docker env variables that
# will be transformed into hadoop config keys
```